cumulusci/tasks/release_notes/parser.py

```diff
@@ -283,3 +284,35 @@ def _add_issue_comment(self, issue):
                 break
         if not has_comment:
             issue.create_comment("{} {}".format(comment_prefix, version_str))
+
+
+class InstallLinkParser(BaseChangeNotesParser):
+    prod_url_template = (
+        "https://login.salesforce.com/packaging/installPackage.apexp?p0={}"
+    )
+
+    test_url_template = (
+        "https://test.salesforce.com/packaging/installPackage.apexp?p0={}"
+    )
+
+    def __init__(self, release_notes_generator, title):
+        super().__init__(title)
+        self.release_notes_generator = release_notes_generator
+
+    def parse(self, change_note):
+        # There's no need to parse lines, this parser gets its values from task options
+        pass
```

Comment:
What about this scenario:
1. release notes are generated during the release_beta flow, including the installation links
2. a release engineer manually runs the task to update the release notes, without specifying the version_id option.

In this case, shouldn't we preserve the existing content rather than removing it?

Reply:
Good point.  Yes, we should preserve the install links section if it's already there and no version id is specified